src/main/java/org/commcare/formplayer/services/MenuSessionRunnerService.java

```diff
@@ -101,15 +101,16 @@
     private static final Log log = LogFactory.getLog(MenuSessionRunnerService.class);
 
     public BaseResponseBean getNextMenu(MenuSession menuSession) throws Exception {
-        return getNextMenu(menuSession, null, 0, "", 0, null);
+        return getNextMenu(menuSession, null, 0, "", 0, null, 0);
```

Comment:
Suggestion: There's a lot of data being passed one arg at a time here, it might make sense moving forward to create a SearchContext Object which just wraps all of these parameters. That will let us set all of these defaults and values in one place,  and prevent ongoing creep / expansion of the args list.

Reply:
Noted. Will keep in mind for any next work around this.